vendor/dell/s6000-hw/etc/modules-load.d/opx-hw.conf

```diff
@@ -1,2 +1,77 @@
+# Generic Modules required by Dell Platforms
+
+i2c_dev
+zram
+i2c-mux-gpio
+i2c-mux-pca9541
+i2c-mux-pca954x
```

Comment:
i2c-mux-pca954x should include support for 9541, so both not required; needed for all platforms except S6000